instances/atlas/parcellationEntity/JBA/JBA_metencephalon.jsonld

```diff
@@ -0,0 +1,13 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_metencephalon",
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": null,
```

Comment:
missing parent

Reply:
In our hierarchy, metencephalon (& telencephalon) are root structures. 

What would the parents be?

Reply:
as stated in the general comment: The brain is the root structure. Therefore I would rename JBA_overall to JBA_brain and then group all GabMaps and telencephalon, metencephalon and diencephalon below it.